portable/GCC/MicroBlazeV9/port.c

```diff
@@ -348,7 +360,15 @@ int32_t lReturn;
 
 	if( lReturn == pdPASS )
 	{
-		lReturn = XIntc_Connect( &xInterruptControllerInstance, ucInterruptID, pxHandler, pvCallBackRef );
+		if( pxHandler )

```

Comment:
Instead of overloading `xPortInstallInterruptHandler`, should we provide a separate API `xPortRemoveInterruptHandler`?

Also, this seems unrelated and should probably be in a separate PR?

Reply:
I prefer passing `NULL` over having separate install/remove methods.  And yes, it's somewhat unrelated, but it's a fairly small change in exactly the same port so it seemed simpler to roll it in together.

Reply:
Agree that it is a small change but I'd still prefer to keep it in separate PR. I am happy to move this part to a new PR for you. Let me know if you would like me to.

Reply:
I addressed this so that we can merge this PR.

Reply:
I also created this PR to merge the other change - https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/533

Please take a look.